### DISCUSSION TITLE
Guy with username I want that hasent done anything sence he made his profile in 2017 (6y ago)

### DISCUSSION THREAD

**Discussion by M1noa** [community/community#51910] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

The GitHub account "[Minoa](https://github.com/minoa)" has the username "Minoa," which I would like to use. However, the account has not been active since it was created in 2017 and has no public repositories. "Minoa" is my real name, and having it as my username on GitHub would be beneficial. I'm wondering if there's anything I can do to acquire the username since the current account holder has not used it for anything. Thanks  -Minoa

**Comment by meppu** [comment]
From [GitHub Username Policy](https://docs.github.com/en/site-policy/other-site-policies/github-username-policy)

> Keep in mind that not all activity on GitHub is publicly visible; accounts with no visible activity may be in active use.

> If the username you want has already been claimed, consider other names or unique variations. Using a number, hyphen, or an alternative spelling might help you identify a desirable username still available.

Also when you select "Dormant, inactive, or squatting username" on [GitHub Support Request](https://support.github.com/contact/account):

![image](https://user-images.githubusercontent.com/112294217/229701981-3250bbef-a47e-47a8-883f-f22c661086cc.png)

**Reply by MuhammadUwais** [reply]
Thanks for the info.

I am also trying to change my username. Using the signup feature I checked for the username that I want and it says "Username is not available" but when I went for `https://github.com/usernameIwant` it gives a 404 error. Does it mean the account no longer exists? And is it possible to contact support and get that username?

**Reply by meppu** [reply]
The account you are looking for probably deleted for some reason, and username will be avaible with in 90 days.

**Reply by ghost** [reply]
> The account you are looking for probably deleted for some reason, and username will be avaible with in 90 days.

Thats not always the case. The account could also be flagged, meaning it can still be used with limited features, but it is not visible to GitHub users. 

@MuhammadUwais

**Comment by ktauchathuranga** [comment]
you cant do that!
unless u contacted him and asked him to change username or delete his profile i think!

**Reply by landsman** [reply]
How can somebody contact a user on GitHub if there are no direct messages available? 
Most users like this do not have visible contact information.

**Reply by rromulos** [reply]
sounds like when you want to buy a domain and it's already taken by someone else. Just choose another name and that's it.
